anaconda_updates/anaconda_updates/releases/f35.py

```diff
@@ -0,0 +1,11 @@
+from . import GeneralBranch, Branch
+
+
+class Fedora35Branch(GeneralBranch):
+
+    def __init__(self):
+        super().__init__(branch_type=Branch.fedora35,
+                         cmd_args=["-f35", "--fedora35"],
+                         help="working on Fedora 35",
+                         version_script_params=["-f35", "-p"],
+                         site_packages="./usr/lib/python3.9/site-packages/")
```

Comment:
I have checked that and it is 3.10. The support for master needs to be updated as well.

Reply:
Thank you, I missed that. For the record, the values here are actually not used for anaconda, so things worked just fine...